cluster/images/hyperkube/Dockerfile

```diff
@@ -78,3 +75,7 @@ RUN ln -s /hyperkube /apiserver \
  && ln -s /hyperkube /kubelet \
  && ln -s /hyperkube /proxy \
  && ln -s /hyperkube /scheduler
+
+# Copy the hyperkube binary
+COPY hyperkube /hyperkube
```

Comment:
(Comment didn't get added here correctly...) It significantly cuts down on rebuild time. Adding it in the beginning
invalidates the entire cache and forces you to go through all of apt-get
commands again.